zipline/pipeline/loaders/blaze/events.py

```diff
@@ -56,6 +56,7 @@ class BlazeEventsLoader(PipelineLoader):
     If the '{TS_FIELD_NAME}' field is not included it is assumed that we
     start the backtest with knowledge of all announcements.
     """
+    dataset = None
```

Comment:
We probably want to call this `default_dataset` because the constructor still needs to accept this as an argument.